eth-types/src/lib.rs

```diff
@@ -398,6 +398,9 @@ pub struct ResultGethExecTrace {
 /// before the step is executed.
 #[derive(Deserialize, Serialize, Clone, Debug, Eq, PartialEq)]
 pub struct GethExecTrace {
+    /// True when the tx could not execute
+    #[serde(default = "default_bool")]
```

Comment:
Any specific reason to do `default_bool` for `invalid`? The other values like `failed` don't do it so just wondering.

Reply:
GethExecTrace is being used to deserialize the rpc response of 'debug_traceBlockByNumber'. The response doesn't have invalid. 

If we don't use `#[serde(default = "default_bool")]`, deserialize will fail.